### DISCUSSION TITLE
Customizing kernel_arm64_load and kernel_arm64_entry values in flat-image-tree.py for different platforms

### DISCUSSION THREAD

**Discussion by brandonchuang** [dentproject/dentOS#266] [Q&A]
Hi everyone,

I have a question regarding the kernel_arm64_load and kernel_arm64_entry values in the flat-image-tree.py script, 
which can be found at this URL: 
https://github.com/dentproject/dentOS/blob/main/tools/flat-image-tree.py#L307

I would like to customize these values for different platforms.
As far as I know, flat-image-tree.conf is not available for every platform, 
and I'm wondering if there's a way to make these values platform-specific or configurable so that I can set them differently for each platform.

Any insights or guidance on how to achieve this would be greatly appreciated. Thank you!

Best regards,
Brandon

**Comment by paulmenzel** [comment]
Dear Brandon,


Thank you for your question. I do not have the answer, but before starting to think about customization per platform, can you please elaborate why you need to change it per platform and give an example?


Kind regards,

Paul

**Comment by brandonchuang** [comment]
Hi Paul,

On as4224 and as4564 accton arm64 platform, the entry is 
'kernel_arm64_load' : "<0x80080000>", 'kernel_arm64_entry' : "<0x80080000>",

But we have another new platform which entry is
'kernel_arm64_load' : "<0x84080000>", 'kernel_arm64_entry' : "<0x84080000>",

That's why I need per platform configuration.

Thanks,
Brandon
